src/cascadia/WpfTerminalControl/TerminalControl.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl x:Class="Microsoft.Terminal.Wpf.TerminalControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:Microsoft.Terminal.Wpf"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+    <Grid>
+        <local:TerminalContainer x:Name="termContainer" Margin="0,0,17,0"/>

```

Comment:
Why 17?

Reply:
17 is the size of scrollbars in wpf. Thinking about it though I think I can get the layout to happen correctly without the magic numbers.